diff --git a/pyproject.toml b/pyproject.toml
index cea04d3..5f0b041 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -16,11 +16,11 @@ python-nix-template = "python_nix_template:main"
 build-backend = "hatchling.build"
 requires      = ["hatchling"]
 
-# [tool.uv.sources]
-# pnt-functional = { workspace = true }
+[tool.uv.sources]
+pnt-functional = { workspace = true }
 
-# [tool.uv.workspace]
-# members = ["packages/*"]
+[tool.uv.workspace]
+members = ["packages/*"]
 
 [tool.hatch.build.targets.wheel]
 packages = ["src/python_nix_template"]
@@ -88,19 +88,19 @@ hatchling = "*"
 
 [tool.pytest.ini_options]
 # Add source locations to pytest-cov e.g.:
-# --cov=packages/pnt-functional/src/pnt_functional/
 addopts = """
 -rA
 --xdoctest
 --disable-warnings
 --cov=src/python_nix_template/
+--cov=packages/pnt-functional/src/pnt_functional/
 --cov-report=term-missing
 --durations=0
 """
 log_level = "INFO"
 testpaths = [
   # Add test locations e.g.:
-  # "packages/pnt-functional/src/pnt_functional/tests",
+  "packages/pnt-functional/src/pnt_functional/tests",
   "src/python_nix_template/tests",
 ]
 
